components/browser/menu/src/main/java/mozilla/components/browser/menu/item/WebExtensionBrowserMenuItem.kt

```diff
@@ -41,10 +41,10 @@ class WebExtensionBrowserMenuItem(
 
     @Suppress("TooGenericExceptionCaught")
     override fun bind(menu: BrowserMenu, view: View) {
-        val imageView = view.findViewById<ImageView>(R.id.action_image)
-        val labelView = view.findViewById<TextView>(R.id.action_label)
-        val badgeView = view.findViewById<TextView>(R.id.badge_text)
-        val container = view.findViewById<View>(R.id.container)
```

Comment:
Nit: Is there any specific reason for the change? Performance or other reason? :)

Reply:
Nothing lol
Just looks nice to me :)

Reply:
😄  I think if we don't have a strong reason for the change, I would prefer to not change it? What do you think? :)

Reply:
Maybe our reason could be "this code is more like 'kotlin-style'" lol
But if you want i can take back changes.